### PR TITLE
update to akka 2.6.14

### PR DESCRIPTION
**Pull request by an-tex**
I've just did the bare minimum to get it to compile & tests passing. I can't remember the full process though, is it necessary to check all js/* files whether the upstream has changed?

### CODE REVIEW

**Comment by an-tex** [comment]
Hey @andreaTP . Any chance you can have a quick at look at this and give further guidance? Would be great to get lagom js to it's latest version too :) Cheers

File: akka-js-actor-stream/js/src/main/scala/akka/stream/Attributes.scala [hunk 1]
```diff
@@ -0,0 +1,875 @@
+/*
```
Comment (andreaTP) [inline akka-js-actor-stream/js/src/main/scala/akka/stream/Attributes.scala:1]: Can you please mark with a comment where this file has been changed from upstream?

**Review by andreaTP — APPROVED** [review]
One single comment and LGTM! thanks a lot @an-tex !

File: akka-js-actor-stream/js/src/main/scala/akka/stream/Attributes.scala (hunk 1 above)
Comment (an-tex) [reply akka-js-actor-stream/js/src/main/scala/akka/stream/Attributes.scala:1]: Thanks for the fast reply! :) I've added comments in the same style as I found it in other files with `// SCALA.JS ...`

**Comment by andreaTP** [comment]
LGTM!
I'll try to push a release by end of the week @an-tex

**Comment by andreaTP** [comment]
Just to make sure you get the notification, @an-tex thanks, 2.2.6.14 is released!

**Pull request merged**